src/main/java/ch/epfl/biop/wrappers/stardist/Stardist.java

```diff
@@ -36,11 +39,12 @@ public static void setStardistEnvType(String stardistEnvType) {
 
     static void execute(List<String> options, Consumer<InputStream> outputHandler) throws IOException, InterruptedException {
         List<String> cmd = new ArrayList<>();
+        List<String> start_cmd = null ;
 
         // Get the prefs about the env type
         String stardistEnvDirectory = Prefs.get(keyPrefix + "Stardist_envDirPath", Stardist.stardistEnvDirectory);
         String stardistEnvType = Prefs.get(keyPrefix + "Stardist_envType", Stardist.stardistEnvType);
-
+/*
```

Comment:
Get rid of the commented code if it's not used anymore. We still have it in the git history if necessary anyway

Reply:
done